OpTestLogger.py

```diff
@@ -0,0 +1,102 @@
+
+# This implements all the python logger setup for op-test
+
+import os
+import sys
+from datetime import datetime
+import logging
+from logging.handlers import RotatingFileHandler
+
+class OpTestLogger():
+    '''
+    This class is used as the main global logger and handler initialization module.
+
+    See common/HelloWorld.py as an example for the usage within an op-test module.
```

Comment:
I think this one is `testcases/HelloWorld.py`

Reply:
Done